README.md

```diff
@@ -33,40 +33,142 @@ Clover Go supports two types of card readers: a magnetic stripe, EMV chip-and-si
+
+1. Go to your **App**’s **Settings** on the [Sandbox Dev Dashboard](https://sandbox.dev.clover.com/developers/). Make sure to save your **App ID** and **App Secret** somewhere; you’ll need them for later.
+![oauth1](/images/oauth1.png)
+
+2. Change the App Type to be REST Clients > **Web**.
+![oauth2](/images/oauth2.png)
+
+3. Make sure that your app has *disabled* Customer Read/Write, Employees Write and enabled the rest of the Permissions in **App Settings** > **Requested Permissions**.
+![oauth3](/images/oauth3.png)
+
+   **Warning**: Failure to set these permissions accordingly will lead to an invalid access token, which will prevent the Clover Go SDK from being initialized.
+
+4. Edit your app’s REST Configuration. The Site URL should be your app’s URL. But if you don’t have one set up yet, you can just use `https://sandbox.dev.clover.com` for now. Make sure the Default OAuth Response is **CODE**.
+![oauth4](/images/oauth4.png)
+
+   **Note**: The developer portal does not currently accept non-http(s) URL schemes. If you have a custom URL scheme for native iOS and Android applications (such as myPaymentApp://clovergoauthresponse), send an email to `Clovergo-Integrations@firstdata.com` with your App ID and redirect URL request.
+
+5. Click the **Market Listing** tab and then on **Preview in App Market**.
+![oauth5a](/images/oauth5a.png)
+
+   **Preview In App Market** opens the app preview page as your test merchant:
+![oauth5b](/images/oauth5b.png)
+
+6. If the app is not installed for your test merchant, click **Connect** and then **Accept** to install the app. If the app is installed, click **Open App**. Either of these steps will open a browser tab with a URL containing a **CODE** parameter: 
+
+   `https://sandbox.dev.clover.com/?merchant_id={MERCHANT_ID}1&employee_id={EMPLOYEE_ID}&client_id={CLIENT_ID}&code={CODE}`
+![oauth6](/images/oauth6.png)
+
+   Save the **CODE** value somewhere.
+
+7. Pass in the **App ID**, **App Secret**, and **CODE** you saved earlier into the following URL: 
+
+   `https://sandbox.dev.clover.com/oauth/token?client_id={APP_ID}&client_secret={APP_SECRET}&code={CODE}`
+
+   (do not include the curly braces).
+
+8. Visit that URL in your browser, and you should be provided with your access token 🎉. 
+
+   **Note**: If you get an “Unknown Client ID” message, check that you don’t include any spaces in the URL and visit the URL again.
+
+### Running the Sample App
+
+#### Prerequisites
+1. Clone the SDK, go into the Example folder and check out the branch for Swift 5:
+
+       git clone https://github.com/clover/remote-pay-ios-go.git
+       cd remote-pay-ios-go/Example
+       git checkout CloverGo_Swift5.0
+2. Ensure that the following dependencies in your `Podfile` are set to the right versions:
+
+       pod 'Starscream', :git => 'https://github.com/daltoniam/Starscream.git', :tag => '3.0.5'pod 'GoConnector', '3.3.5'
```

Comment:
if its 3.3.5 we should be asking them to use Xcode 11.1 and set Swift 5 as discussed

Reply:
Have made this more explicit.